attributes

```diff
@@ -0,0 +1,95 @@
+[nextcloudUser]
+Type=ExtendedOption
+ObjectClass=nextcloudUser
+Module=users/user
+Default=1
+Editable=1
+Description=Nextcloud
+DescriptionDE=Nextcloud
+LongDescription=User can access Nextcloud
```

Comment:
seems now seems to collide with the description of the next item?

Reply:
@blizzz The checkbox is not shown anymore in the UMC, so the description is not displayed.